packages/account/src/test-utils/launchNode.ts

```diff
@@ -198,10 +198,12 @@ export const launchNode = async ({
       const chainConfigWritePath = path.join(tempDirPath, 'chainConfig.json');
       const stateConfigWritePath = path.join(tempDirPath, 'stateConfig.json');
       const metadataWritePath = path.join(tempDirPath, 'metadata.json');
+      const stateTransitionWritePath = path.join(tempDirPath, 'state_transition_bytecode.wasm');
 
       writeFileSync(chainConfigWritePath, JSON.stringify(chainConfigJson), 'utf8');
       writeFileSync(stateConfigWritePath, fixedStateConfigJSON, 'utf8');
       writeFileSync(metadataWritePath, JSON.stringify(metadataJson), 'utf8');
+      writeFileSync(stateTransitionWritePath, JSON.stringify(''));
```

Comment:
@petertonysmith94 I am just creating this file without adding anything to it.

Starting on [fuel-core 0.25](https://github.com/FuelLabs/fuel-core/pull/1826) we need the file `state_transition_bytecode.wasm` within the snapshot directory. 

Without this file, `fuel-core` does not start.

Since we are going to use it right now, is does not need any data within it, but it needs to be there whenever we are going to run `fuel-core`